sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVOptions.scala

```diff
@@ -90,6 +90,7 @@ private[csv] class CSVOptions(@transient private val parameters: Map[String, Str
   val permissive = ParseModes.isPermissiveMode(parseMode)
 
   val nullValue = parameters.getOrElse("nullValue", "")
+  val emptyValue = parameters.getOrElse("emptyValue", "")
```

Comment:
Yes, null and empty can not be differentiated when they are set to same value.  Currently null value check has higher precedence than the empty value. 

input.csv
1,
2,”” 

Output will be: 
1, null
2, null


I think this behavior  is ok.  By default  Univocity CSV parser  used in spark  also returns  null for empty strings.

I agree we should document this behavior.